widgetastic_patternfly.py

```diff
@@ -618,7 +618,7 @@ def select_by_visible_text(self, *items):
                 './div/ul/li/a[./span[contains(@class, "text") and normalize-space(.)={}]]'.format(
                     quote(text)),
                 parent=self)
-        self.close()
+        # self.close()
```

Comment:
Okay, so that is because the select disappears. So, in this case, instead of removing `close`, which is usually required, I would change the `close` method to catch `NoSuchElementException` and bail out silently, with a logger notice (something like "Not closing the select %r because it is no longer present on the page"). What is your opinion on that?

@psav @mshriver @RonnyPfannschmidt @izapolsk  maybe you can voice your opinion in?